frontend/shared/components/src/events/EditEventView.vue

```diff
@@ -321,6 +326,23 @@ const { externalOrganization, choose: chooseOrganizer } = useExternalOrganizatio
     }),
 );
 
+const eventGroupOrganization = computed(() => {
+    if (patched.value.group === null) {
+        return null;
+    }
+
+    // prevent double requests
+    if (patched.value.organizationId === patched.value.group.organizationId) {
+        return externalOrganization.value;
+    }
+
+    return useExternalOrganization(
+        computed(() => patched.value.group?.organizationId ?? null),
+    ).externalOrganization.value;
+});
```

Comment:
Preventing duplicate requests is the responsibility of useExternalOrganization + never use hooks inside computed

Reply:
You can pass the organizationHint